tools/distpartitioning/parmetis_wrapper.py

```diff
@@ -0,0 +1,110 @@
+import os
+import sys
+import argparse
+import logging
+import platform
+import constants
+from pathlib import Path
+from utils import read_json
+
+def run_parmetis_wrapper(params):
+    """Function to execute all the steps needed to run ParMETIS
+
+    Parameters:
+    -----------
+    params : argparser object
+        an instance of argparser class to capture command-line arguments
+    """
+    assert os.path.isfile(params.schema_file)
+    assert os.path.isfile(params.hostfile)
+
+    parmetis_install_path = None
+    if params.parmetis_install_path is not None:
+        parmetis_install_path = params.parmetis_install_path
+        if parmetis_install_path[-1] != "/":
+            parmetis_install_path += "/"
+
+    schema = read_json(params.schema_file)
+    graph_name = schema[constants.STR_GRAPH_NAME]
+    num_partitions = len(schema[constants.STR_NUM_NODES_PER_CHUNK][0])
+
+    #Trigger pre-processing step to generate input files for ParMETIS
+    env = dict(os.environ)
+    dgl_home = env["DGL_HOME"]
+    if dgl_home[-1] != "/":
+        dgl_home += "/"
+    logging.info(f'DGL Installation directory: {dgl_home}')
+    preproc_cmd = f'mpirun -np {num_partitions} -hostfile {params.hostfile} '\
+                  f'python3 {dgl_home}tools/distpartitioning/parmetis_preprocess.py '\
+                  f'--schema_file {params.schema_file} '\
+                  f'--output_dir {params.preproc_output_dir}'
+    logging.info(f'Executing Preprocessing Step: {preproc_cmd}')
+    os.system(preproc_cmd)
+    logging.info(f'Done Preprocessing Step')
+    logging.info("\n")
```

Comment:
why do we need logging.info("\n") x 3?

Reply:
Its just to highlight that each step is done correctly and completed properly. So that the user notices that each step is completed. If this is not needed, I will remove it. Removed.